tools/travis_build_and_test.sh

```diff
@@ -24,6 +24,11 @@ RESULT=0
 pip uninstall -y numpy
 pip install -r requirements.txt
 
+# Bring in necessary ROM data files
+mkdir -p ${PWD}/rom_data
+rsync --exclude='SEOBNRv1ROM*' --exclude='SEOBNRv2ROM_DS_HI_v1.0.hdf5' --exclude='NRSur4d2s_FDROM.hdf5' -ravz pycbc@sugwg-condor.phy.syr.edu:/cvmfs/oasis.opensciencegrid.org/ligo/sw/pycbc/lalsuite-extra/${LAL_EXTRA_TAG}/share/lalsimulation/ ${PWD}/rom_data
```

Comment:
Don't do it this way. You'll need to do something similar to what Bernd does here:

https://github.com/ligo-cbc/pycbc/blob/master/tools/einsteinathome/pycbc_build_eah.sh#L1282

Remember that Travis disk space is very limited, so you should only bring in the minimal set of ROM data.